cli.ts

```diff
@@ -176,7 +176,11 @@ let refreshToken;
 /// //////////////////////////////////////////////////////////////////////
 // REQUEST HANDLING
 
-function responseToError(response) {
+function responseToError(response, error) {
+  if (response === undefined) {
```

Comment:
I think we can safely broaden that check to 
```suggestion
  if (!response) {
```